Repository/AbstractPhpcrRepository.php

```diff
@@ -147,6 +148,15 @@ public function add($path, $resource)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function reorder($sourcePath, $position)
+    {
+        Assert::greaterThanEq($position, 0, 'Reorder position cannot be negative, got: %s');
+        $this->reorderNode($sourcePath, $position);
```

Comment:
All the other abstract methods refer to nodes, and we are ultimately
moving the PHPCR node, can change, but this isn't public API so I am
happy to refer to nodes here.

On Mon, Jun 27, 2016 at 12:58:46AM -0700, David Buchmann wrote:
>    In [1]Repository/AbstractPhpcrRepository.php:
> 
>  > @@ -147,6 +148,15 @@ public function add($path, $resource)
>  >      }
>  >
>  >      /**
>  > +     * {@inheritdoc}
>  > +     */
>  > +    public function reorder($sourcePath, $position)
>  > +    {
>  > +        Assert::greaterThanEq($position, 0, 'Reorder position cannot be negative, got: %s');
>  > +        $this->reorderNode($sourcePath, $position);
> 
>    is reorderNode the right name? it could be doReorder maybe? on odm its a
>    document, not a node that we reorder.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly, [2]view it on GitHub, or [3]mute the thread.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource/pull/28#discussion_r68535517
>    2. https://github.com/symfony-cmf/resource/pull/28/files/6bb50cfa74bf87e110e9b44c5c4a639b98ef8f37#r68535517
>    3. https://github.com/notifications/unsubscribe/AAgZcWIjLr-DqaHJn4Cgmjv49VKsCjobks5qP4K2gaJpZM4I9hWA
>    4. https://github.com/symfony-cmf/resource/pull/28/files/6bb50cfa74bf87e110e9b44c5c4a639b98ef8f37#r68535517